recipe/meta.yaml

```diff
@@ -68,7 +78,11 @@ outputs:
         - setuptools
         - cython
         - six >=1.6.0
-        - abseil-cpp 20211102.0
+        - libabseil                   # [unix]
+        # windows not ready for C++17 of shared abseil builds yet;
+        # keep one line unpinned for migrator to pick it up
+        - libabseil-static            # [win]
```

Comment:
If you are linking statically, you need to redistribute the license of abseil in this packge.

Reply:
Thanks, done.